src/org/jitsi/impl/neomedia/MediaStreamStatsImpl.java

```diff
@@ -1209,13 +1209,12 @@ private void setRttMs(long rttMs)
             {
                 // RemoteBitrateEstimator is a CallStatsObserver and
                 // VideoMediaStream has a RemoteBitrateEstimator.
-                MediaStream mediaStream = this.mediaStreamImpl;
+                MediaStreamImpl mediaStream = this.mediaStreamImpl;
```

Comment:
Thinking out loud a bit here...

This defeats the point of the interface a bit, doesn't it?  If classes have methods that aren't in the interface, you have to cast down to them which means you have to care about which specific implementation you're getting.  Sometimes that makes sense (VideoChannel having different methods than AudioChannel, for example, and at some point a callee getting passed a VideoChannel type specifically), but in my head it feels odd for MediaStream and MediaStreamImpl.  The more places we downcast the less flexibility we have to use a different impl there, and I'm assuming there's value in that flexibility since we created the interface at all, but maybe that isn't really true?  Are there other implementations of MediaStream that might differ here?